doc/optimizers/pysnopt.rst

```diff
@@ -16,8 +16,9 @@ Installation
 SNOPT is available for purchase `here 
 <http://www.sbsi-sol-optimize.com/asp/sol_snopt.htm>`_. Upon purchase, you should receive a zip file. Within the zip file, there is a folder called ``src``. To use SNOPT with pyoptsparse, paste all files from ``src`` except snopth.f into ``pyoptsparse/pySNOPT/source``.
 
-From v2.0 onwards, only SNOPT v7.7 is officially supported.
+From v2.0 onwards, only SNOPT v7.7.x is officially supported.
 To use pyOptSparse with previous versions of SNOPT, please checkout release v1.2.
+The current version of SNOPT being tested is v7.7.1, although v7.7.5 is also expected to work.
```

Comment:
Can you explain what you mean by this? I saw that  you updated the Docker image to use 7.7.5.
Should we check this PR with both versions?

Reply:
Oh yeah good point, I wrote this before deciding that we should get Docker to build with 7.7.5. From a CI point of view, I don't want to test multiple SNOPT versions, so I guess we've settled on 7.7.5 for that. For manually testing this PR, yes you should test with both versions and that's what I did.

I will update the documentation to say we test 7.7.5.